core/src/main/scala/filodb.core/memstore/TimeSeriesShard.scala

```diff
@@ -95,6 +95,15 @@ class TimeSeriesShardStats(dataset: DatasetRef, shardNum: Int) {
   val evictedPartKeyBloomFilterFalsePositives = Kamon.counter("evicted-pk-bloom-filter-fp").refine(tags)
   val evictedPkBloomFilterSize = Kamon.gauge("evicted-pk-bloom-filter-approx-size").refine(tags)
   val evictedPartIdLookupMultiMatch = Kamon.counter("evicted-partId-lookup-multi-match").refine(tags)
+
+  /**
+    * Difference between the local clock and the received ingestion timestamps, in milliseconds.
+    * If this gauge is negative, then the received timestamps are ahead, and it will stay this
+    * way for a bit, due to the monotonic adjustment. When the gauge value is positive (which is
+    * expected), then the skew reflects the delay between the generation of the samples and
+    * receiving them, assuming that the clocks are in sync.
+    */
+  val ingestionClockSkew = Kamon.gauge("ingestion-clock-skew").refine(tags)
```

Comment:
Could we call this `ingestion-clock-delay`?  Typically we think of delay as how far behind real time we are, which is pretty much what this measures.  I think of skew as more like difference in timestamps between different nodes/instances in the same cluster... or I guess between apps.

Reply:
done